packages/jest-circus/src/utils.ts

```diff
@@ -299,8 +300,13 @@ export const makeSingleTestResult = (
 
   let location = null;
   if (includeTestLocationInResult) {
-    const stackLine = test.asyncError.stack.split('\n')[1];
-    const parsedLine = stackUtils.parseLine(stackLine);
+    const stackLines = test.asyncError.stack.split('\n');
+    const stackLine = stackLines[1];
+    let parsedLine = stackUtils.parseLine(stackLine);
+    if (parsedLine?.file?.includes(`${path.sep}jest-each${path.sep}`)) {
```

Comment:
should we have some sort of
```ts
const jestEachBaseDir = path.dirname(require.resolve('jest-each'));
if (parsedLine?.file?.startsWith(jestEachBaseDir))
```
?

the const could be in the module scope

Reply:
That's a good idea. I'm trying to think if that'll cause any issues with linked dependencies or anything but I don't think it will and it's safer than my current method. Only risk is if `jest-each` changes to some super weird structure where the main file is nested inside a subdirectory and dependent modules are in parent directories, but that seems really unlikely and will be caught by tests.